client/public/index.html

```diff
@@ -8,6 +8,7 @@
   <title> Andela Socials </title>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
   <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Abel" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
```

Comment:
If the only icon we are using from this library is the `Slack` logo icon, I think it would be better to simply download the Slack logo icon as SVG and then add it to the `client/assets/icons` folder as a loadable component. That way, our client will not need to download the entire `fontawesome` library just to use 1 icon.

Reply:
Alright

Reply:
Thanks @samdiano. Looks good.